src/bootstrap/compile.rs

```diff
@@ -722,6 +722,9 @@ impl Step for Rustc {
                     cargo.rustflag("-Cembed-bitcode=yes");
                 }
                 RustcLto::ThinLocal => { /* Do nothing, this is the default */ }
+                RustcLto::Off => {
```

Comment:
Just FYI, this doesn't disable LTO for the compiler built in stage0.  I think this would need to go outside the check for the compiler stage.